frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -13,29 +13,33 @@ import { EMPTY_OBJECT } from "../constants/utils";
 import { DEFAULT_FETCH_OPTIONS, JSON_BODY_FETCH_OPTIONS } from "./common";
 import { ENTITIES } from "./entities";
 
+interface OntologyTerm {
+  [id: string]: string;
+}
+
+interface OntologyTermsByOrganism {
+  [organismId: string]: Array<OntologyTerm>;
+}
 interface RawPrimaryFilterDimensionsResponse {
-  gene_terms: { [organismId: string]: Array<{ [id: string]: string }> };
-  organism_terms: { [id: string]: string }[];
+  gene_terms: OntologyTermsByOrganism;
```

Comment:
Rename `OntologyTermsByOrganism` to `RawOntologyTermsByOrganism` so we preserve flatten keyword for actually flattened array.